index.js

```diff
@@ -255,6 +255,7 @@ const D2LHypermediaClasses = {
 	rubrics: {
 		analytic: 'analytic',
 		assessmentCriterionCell: 'assessment-criterion-cell',
+		associations: 'associations',
```

Comment:
Is this correct to be plural?

Reply:
Yes. It refers to a set of associations.

Reply:
Will there also be one for `association` then? Similar to `outcomes` with `intent`/`intents`.